types/places.js/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for algolia places 1.16
+// Project: https://github.com/algolia/places
+// Definitions by: Ghizlane LOTFI <https://github.com/ghizlanelotfi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
+
+import {EventEmitter} from 'events';
+
+declare module 'places.js' {
```

Comment:
You need to unwrap the contents of this module and remove the `declare module 'places.js'` entirely.